Sources/libhostmgr/Model/Paths.swift

```diff
@@ -44,6 +44,10 @@ public extension Paths {
         storageRoot.appendingPathComponent("git-mirrors", isDirectory: true)
     }()
 
+    static let botUserSshKeyDirectory: URL = {
+        storageRoot.appendingPathComponent("bot-user-ssh-key", isDirectory: true)
```

Comment:
Nitpick but I'd suggest naming this shared path something more like `shared-ssh-keys` or `shared-credentials` or similar:
 - We have more than one SSH keys, so: plural
 - In case that, in the future, we might want to use that shared drive to share more than just the very specific case of bot ssh keys (e.g. API tokens, `.netrc` for `pod trunk push`, or whatnot)

Reply:
Oh good point, nice suggestions 👍 . While we added this security layer for SSH keys, I was thinking we could potentially do something similar for some API tokens (e.g. the GitHub one).

Reply:
My problem with `shared` in the name is that this is kinda implicit (it will be in the shared volume). What about `bot-users-credentials`, `bot-credentials` or  even just `bot-keys`? 🤔

Reply:
Yeah `bot-keys` would work.
Would not scale super well if we later extend the use of this shared volume to store e.g. [API token for `gem push`](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/.buildkite/commands/gem-push.sh#L11-L13) or similar though…
Naming is hard 😅

Reply:
> My problem with shared in the name is that this is kinda implicit (it will be in the shared volume)

Also, my thinking of the `shared` word in the name was not really about it being shared between the host and the VM (like the `Shared` in `Shared Volume` means), but instead about those bot keys being shared by multiple repos. Maybe using a term like `common` would avoid the confusion in that regard?

Reply:
> but instead about those bot keys being shared by multiple repos

Ahh got it. Thinking about it, it doesn't sound that bad, but it would still be nice to avoid the confusion. I think I'll start with `common-credentials` (and we can still change it before shipping the final images for this change). Updated on 604978e361e7451bfe71b69545489a884d4977f7.